src/components/CloudHeader/CloudHeaderContainer.js

```diff
@@ -0,0 +1,25 @@
+import PropTypes from 'prop-types';
+import React from 'react';
+import classNames from 'classnames';
+
+const CloudHeaderContainer = props => {
```

Comment:
What does the `Container` notation here denote/correspond to? Feel like there might be potential ambiguity to the connected "container" component associated with Redux-like libraries.

Reply:
You could potentially change it to `CloudHeaderWrapper` if that makes more sense

Reply:
👍